assets/package.json

```diff
@@ -57,7 +57,9 @@
     "typescript": "^3.7.5",
     "uuid": "^8.3.2",
     "vm-browserify": "^1.1.2",
-    "whatwg-fetch": "^3.0.0"
+    "whatwg-fetch": "^3.0.0",
+    "@rjsf/core":"^2.5.1",
```

Comment:
did you manually add these? if you add through `yarn add` then they would be alphabetically ordered, which I believe `@` things end up at the top